content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md

```diff
@@ -13,16 +13,16 @@ weight: 70
 
 {{% capture overview %}}
 
-A _job_ creates one or more pods and ensures that a specified number of them successfully terminate.
-As pods successfully complete, the _job_ tracks the successful completions.  When a specified number
-of successful completions is reached, the job itself is complete.  Deleting a Job will cleanup the
-pods it created.
+A Job creates one or more pods and ensures that a specified number of them successfully terminate.
+As pods successfully complete, the Job tracks the successful completions.  When a specified number
```

Comment:
Do you want to fix Pod casing throughout this doc too?

Reply:
To me this reads better using pod in lowercase where using the term more loosely. It comes down to a matter of style and taste, I think.